azkaban-exec-server/src/main/java/azkaban/execapp/JobRunner.java

```diff
@@ -757,10 +758,10 @@ private void insertJobMetadata() {
       this.props.put(CommonJobProperties.EXECUTION_LINK,
           String.format("%s/executor?execid=%d", baseURL, this.executionId));
       this.props.put(CommonJobProperties.JOBEXEC_LINK, String.format(
-          "%s/executor?execid=%d&job=%s", baseURL, this.executionId, this.jobId));
+          "%s/executor?execid=%d&job=%s", baseURL, this.executionId, this.node.getNestedId()));
       this.props.put(CommonJobProperties.ATTEMPT_LINK, String.format(
           "%s/executor?execid=%d&job=%s&attempt=%d", baseURL, this.executionId,
-          this.jobId, this.node.getAttempt()));
+          this.node.getNestedId(), this.node.getAttempt()));
```

Comment:
Ok, nice. Looks like the UI is already using nestedId:

https://github.com/azkaban/azkaban/blob/4f30ad3b778ec6366251856158d4cc2dc68e7ad4/azkaban-web-server/src/web/js/azkaban/view/flow-execution-list.js#L391-L395

https://github.com/azkaban/azkaban/blob/4f30ad3b778ec6366251856158d4cc2dc68e7ad4/azkaban-web-server/src/web/js/azkaban/view/flow-execution-list.js#L215-L218